.github/workflows/main.yml

```diff
@@ -80,7 +80,7 @@ jobs:
         if: matrix.os == 'windows-latest'
         run: |
           cd extensions/ql-vscode
-          $env:CODEQL_PATH=$(Join-Path $env:GITHUB_WORKSPACE -ChildPath 'codeql-home/codeql/codeql.cmd')
```

Comment:
Although this seems like the right thing to do for now, I'm just a little worried at the divergence of the test environment from the actual user experience.

Reply:
Maybe a comment/issue documenting why CI appears to be quarantining the executable, and what else we speculate we could do about that if something changes about Actions's VM setup in the future, or something?

Reply:
I think we are trying to move away from using the `codeql.cmd` file. If you look at the contents of the file, it is just a thing wrapper around calling the executable. The failing build always had to do with the `exe` file, so moving back to calling `cmd` should have no effect.

We should consider explicitly deprecating (and later removing) the `cmd` file. I'll bring that up as a discussion point.

Reply:
Ah, ok, I'd misunderstood. This is totally fine then.